Source/Core/DolphinQt/Config/GameConfigWidget.h

```diff
@@ -50,8 +51,10 @@ class GameConfigWidget : public QWidget
   ConfigBool* m_use_monoscopic_shadows;
 
   ConfigStringChoice* m_deterministic_dual_core;
-  ConfigSlider* m_depth_slider;
-  ConfigInteger* m_convergence_spin;
+  ConfigFloatSlider* m_depth_slider;
+  ConfigFloatSlider* m_convergence_spin;
+  QLabel* m_depth_slider_value;
+  QLabel* m_convergence_spin_value;
```

Comment:
These `QLabel` don't need to be member variables, they can just be local variables inside `GameConfigWidget::CreateWidgets`.